setup/artifacts/github.md

```diff
@@ -39,6 +41,7 @@ First, make sure that artifact support is enabled:
 
 ```bash
 hal config features edit --artifacts true
+hal config artifact github enable
```

Comment:
Above, it's "First, make sure that artifact support is enabled" which would still be `hal config features edit --artifacts true` right?  Should there be an additional step here for enabling github?